src/main/java/bridge/model/BridgeLength.java

```diff
@@ -0,0 +1,38 @@
+package bridge.model;
+
+import bridge.util.constant.Errors;
+import bridge.util.constant.Numbers;
+
+public class BridgeLength {
+    private final int length;
+
+    public BridgeLength(String length) {
+        validate(length);
+        this.length = Integer.parseInt(length);
+    }
+
+    private static void validate(String length) {
+        //숫자로 이루어져있는 지 구성 → 숫자가 범위 내의 숫자인지
+        isNumber(length);
+        int number = Integer.parseInt(length);
+        validateRange(number);
+    }
+
+    private static  void isNumber(String input) {
+        try{
+            Integer.parseInt(input);
+        } catch (NumberFormatException e) {
+            throw new IllegalArgumentException(Errors.NOT_NUMBER_ERROR.getMessage());
+        }
+    }
```

Comment:
```suggestion
    privat void validate(String length) {
        //숫자로 이루어져있는 지 구성 → 숫자가 범위 내의 숫자인지
        int number = toInt(length);
        validateRange(number);
    }

    privat  int toInt(String input) {
        try{
            return Integer.parseInt(input);
        } catch (NumberFormatException e) {
            throw new IllegalArgumentException(Errors.NOT_NUMBER_ERROR.getMessage());
        }
    }
```

체크겸 전환까지 해서 와도 좋을 것 같아요!